main.go

```diff
@@ -59,14 +59,14 @@ func main() {
 }
 
 func dbOptions(environment string) (postgres.Options, error) {
-	maxConnections, err := getMaxConnections()
+	maxConnections, err := strconv.Atoi(os.Getenv("MAX_DB_CONNECTIONS"))
```

Comment:
Ok just a heads up if someone forgets to set `MAX_DB_CONNECTIONS` in local testing or whatever the server is just going to stay stuck trying to `UpdateEvents` from TBA with zero connections. So we should either change this to require at least one connection, or change the server to default to max of one, and issue a warning if maxConnections is less than one.

Reply:
That will return an error if `MAX_DB_CONNECTIONS` is an empty string which is what it should be if they don't specify `MAX_DB_CONNECTIONS`.

Reply:
Then maybe the server should check the value of maxConnections and use a default value if it is 0 or -3 or something, and issue a message like "Using default valur for max connections of 20" or whatever.